core/Db/Schema/Mysql.php

```diff
@@ -81,6 +81,8 @@ public function getTablesCreateSql()
                           `setting_name` VARCHAR(255) NOT NULL,
                           `setting_value` LONGTEXT NOT NULL,
                           `user_login` VARCHAR(100) NOT NULL DEFAULT '',
+						  `plugin_setting_id` BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
```

Comment:
Would it be possible to make the spacing consistent with the lines above (ie. use spaces not tabs)?

Reply:
Done, tabs were changed to spaces in Mysql.php.